content/en/continuous_delivery/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Continuous Delivery Visibility
+kind: documentation
+further_reading:
+- link: "https://app.datadoghq.com/release-notes?category=CI%20Visibility"
+  tag: "Release Notes"
+  text: "Check out the latest CI/CD Visibility releases! (App login required)"
+- link: "continuous_delivery/deployments"
+  tag: "Documentation"
+  text: "Learn how to set up Deployment Visibility"
+- link: "continuous_delivery/search"
+  tag: "Documentation"
+  text: "Learn how to search and manage your deployment results"
+- link: "/continuous_delivery/explorer"
+  tag: "Documentation"
+  text: "Learn about the CD Visibility Explorer"
+cascade:
+    algolia:
+        rank: 70
+        tags: ['ci/cd', 'continuous delivery']
+---
+
+## Overview
+
+Datadog Continuous Delivery (CD) Visibility unifies information about deployments in your CD providers in addition to data about your environments. CD Visibility brings deployment metrics and data into Datadog dashboards and notebooks so you can communicate the health of your deployments and environments, and focus your efforts in improving your team's ability to deliver quality code every time.
+
+// Vimeo link to CD Visibility demo
+
+CD Visibility enables developers to x, y, and z. Further, it also provides build engineers with visibility into this and that. 
+
+## Improve deployment reliability
```

Comment:
@alai97 let's remove this section for now.  We can create more 'promotional' content once we have messaging ready for the website and other marketing-owned properties.